src/components/Books/AddBook.js

```diff
@@ -0,0 +1,15 @@
+const AddBook = () => (
+  <div className="add-book">
+    <h1>Add Book</h1>
+    <form className="form">
+      <input type="text" placeholder="Enter Book Name" />
+      <input type="submit" />
+      <br />
+      <br />
+      <input type="text" placeholder="Enter Book Name" />
+      <input type="submit" />
+    </form>
```

Comment:
Your form for creating a `new book` should have only one `submit button` according to the project requirement.